setup.py

```diff
@@ -24,23 +24,18 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3.10',
     ],
 
     python_requires='>=3.7',
 
     install_requires=[
         'affine',
         'boto3',
```

Comment:
Since you're not importing/using this directly, you could add a comment string to say `# for rasterio vsis3 support`